psi4/driver/p4util/python_helpers.py

```diff
@@ -520,3 +522,27 @@ def py_psi_set_global_option_python(key, EXTERN):
 
 
 core.set_global_option_python = py_psi_set_global_option_python
+
+
+def core_get_variables():
+    warnings.warn("Using `psi4.core.get_variables` instead of `psi4.core.variables` is deprecated, and in 1.4 it will stop working\n", category=FutureWarning, stacklevel=2)
```

Comment:
Shouldn't these be `DeprecationWarning`s? Might want to YAPF this file as well.

Reply:
I agree that everyone else is using `DeprecationWarning`, and I'm glad to change. But reading [this table and the note at the end](https://docs.python.org/3/library/warnings.html#warning-categories) suggested that `FutureWarning` was right (for targeting users' input files).